usaspending_api/awards/urls.py

```diff
@@ -1,7 +1,23 @@
-from django.conf.urls import url
+from django.conf.urls import include, url
 from usaspending_api.awards import views
 
-urlpatterns = [
+award_id_patterns = [
     url(r'^$', views.AwardList.as_view()),
-    url(r'^summary', views.AwardListSummary.as_view())
+    url(r'^uri/(?P<uri>\w+)', views.AwardList.as_view()),
+    url(r'^fain/(?P<fain>\w+)', views.AwardList.as_view()),
+    url(r'^piid/(?P<piid>\w+)', views.AwardList.as_view())
+]
+
+award_summary_id_patterns = [
+    url(r'^$', views.AwardListSummary.as_view()),
+    url(r'^uri/(?P<uri>.+)', views.AwardListSummary.as_view()),
+    url(r'^fain/(?P<fain>.+)', views.AwardListSummary.as_view()),
+    url(r'^piid/(?P<piid>.+)', views.AwardListSummary.as_view()),
+    url(r'^fy/(?P<fy>[0-9]{4})', views.AwardListSummary.as_view()),
+    url(r'^agency/(?P<agency>.{4})', views.AwardListSummary.as_view())
```

Comment:
I've changed this so you can specify url/?fy=2016 or ?agency=FPDScode or ?agency=FPDScode&fy=2016 on any award summary request